enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/catchup/tx/CatchupPollingProcessTest.java

```diff
@@ -96,7 +102,8 @@ public void shouldSendPullRequestOnTick() throws Throwable
         when( txApplier.lastQueuedTxId() ).thenReturn( lastAppliedTxId );
 
         // when
-        timeoutService.invokeTimeout( TX_PULLER_TIMEOUT );
+        timerService.invoke( TX_PULLER_TIMEOUT );
+        scheduler.runJob();
```

Comment:
These lines are repeated - may be worth a method?

Reply:
Ah, I actually wrote these tests before I introduced the FakeClockJobScheduler which I think can be used here. Will attempt!